drivers/usb_c/vbus/usbc_vbus_tcpci.c

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2023 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT zephyr_usb_c_vbus_tcpci
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(usbc_vbus_tcpci, CONFIG_USBC_LOG_LEVEL);
+
+#include <zephyr/device.h>
+#include <zephyr/kernel.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/usb_c/tcpci_priv.h>
+#include <zephyr/drivers/usb_c/usbc_vbus.h>
+#include <zephyr/drivers/usb_c/usbc_pd.h>
+#include <zephyr/usb_c/tcpci.h>
+#include <zephyr/shell/shell.h>
+
+struct vbus_tcpci_cfg {
+	const struct i2c_dt_spec i2c;
+};
+
+struct vbus_tcpci_data {
```

Comment:
Should this be empty?

Reply:
I left it at the beginning in case I would need some variables. Will remove it.